### ISSUE TITLE
Failed to build under i686-pc-windows-gnu

### ISSUE THREAD

**Issue by zonyitoo** [danburkert/memmap-rs#15]
AppVeyor build log: https://ci.appveyor.com/project/zonyitoo/context-rs/build/job/628poi9n7jr46ga2

Error log:

```
   Compiling memmap v0.1.0
     Running `rustc C:\Users\appveyor\.cargo\registry\src\github.com-121aea75f9ef2ce2\memmap-0.1.0\src\lib.rs --crate-name memmap --crate-type lib -g -C metadata=ea906f06964a25e2 -C extra-filename=-ea906f06964a25e2 --out-dir C:\projects\context-rs\target\debug\deps --emit=dep-info,link -L dependency=C:\projects\context-rs\target\debug\deps -L dependency=C:\projects\context-rs\target\debug\deps --extern bitflags=C:\projects\context-rs\target\debug\deps\libbitflags-d8aad5e08369cb92.rlib --extern winapi=C:\projects\context-rs\target\debug\deps\libwinapi-3afcd2170e6034e1.rlib --extern kernel32=C:\projects\context-rs\target\debug\deps\libkernel32-c606ce60307b6481.rlib --extern libc=C:\projects\context-rs\target\debug\deps\liblibc-165b5479aec0459c.rlib --cap-lints allow`
C:\Users\appveyor\.cargo\registry\src\github.com-121aea75f9ef2ce2\memmap-0.1.0\src\windows.rs:109:57: 109:67 error: mismatched types:
 expected `u32`,
    found `u64`
(expected u32,
    found u64) [E0308]
C:\Users\appveyor\.cargo\registry\src\github.com-121aea75f9ef2ce2\memmap-0.1.0\src\windows.rs:109                                                         len as u64) };
                                                                                                                                                          ^~~~~~~~~~
C:\Users\appveyor\.cargo\registry\src\github.com-121aea75f9ef2ce2\memmap-0.1.0\src\windows.rs:109:57: 109:67 help: run `rustc --explain E0308` to see a detailed explanation
error: aborting due to previous error
Build failed, waiting for other jobs to finish...
Could not compile `memmap`. 
```

**Comment by danburkert** [comment]
Thanks for the report, looking into this now.

**Comment by danburkert** [comment]
Fixed by ae3614084a57f242bee46fde1a46b9f3ff67a12f.  I'll push a new version to crates.io in a bit.

**Comment by danburkert** [comment]
Pushed to crates.io with version [0.2.1](https://crates.io/crates/memmap/0.2.1).
